### ISSUE TITLE
Please consider a second recipe for the pestle

### ISSUE THREAD

**Issue by Mythrrinthael** [RootsTeam/Roots#134]
The wiki page told me I could find all the info I wanted in the tablet. Sure enough, there it is: 5 minutes in and I hit a snag because I removed diorite from worldgen (it's textures make me nauseous). I'm aware you can make some using cobble and quartz, but that's only available well into midgame.

Maybe this second recipe could be a mix of cobblestone and smooth stone?
EDIT: Or perhaps allow any of the non-regular stone to be used? Andesite is the only one I keep due to the other two making me sick (diorite almost literally).

**Comment by elucent** [comment]
Considering the large number of people who have told me that it's a good thing that I give diorite an actual use, I'd rather not. Sorry, but it's not my fault if you remove diorite from your world. 

For you, I'd recommend using CraftTweaker. You can add a new recipe for the pestle yourself using a simple script.

**Comment by Mythrrinthael** [comment]
Huh? Did I miss something? Is JEI wrong to tell me that a lot of your recipes use any of the three special stone types? It seems rather arbitrary how the pestle is diorite-only, compared to how indiscriminate all the other recipes are with special stone. I'm not asking for diorite to be smote from your mod's ingredient list, if that's what you've somehow taken from this. I'm asking for the consistency of other recipes to be dragged on to the pestle.

If nothing else, at least consider adding a blurb on the wiki that you **need** diorite. I don't remove it from world gen for kicks - looking at diorite for longer than a few seconds makes me physically ill. That's not your fault obviously, but

**Comment by elucent** [comment]
It's a white stone pestle, it's made out of white stone. It makes more sense, and it's the recipe I would like for my mod.

You are easily free to change it with the use of CraftTweaker, but a personal issue like this, due to your manipulation of vanilla mechanics and removal of a crucial crafting ingredient, is not something that will cause a change in the mod as a whole.

**Comment by Mythrrinthael** [comment]
If you want it to be the only recipe, then that's that - but _please_ add at least a single sentence on the wiki about the special stone types.

**Comment by elucent** [comment]
What wiki? The in-game documentation? No matter what source you use, that recipe is displayed in exclusively diorite. Even JEI will say "accepts any 'stone'" for any of the recipes that accept any of the vanilla stones -- it won't say that for this recipe.

**Comment by Mythrrinthael** [comment]
[This wiki](http://i.imgur.com/PYIqpkx.png). third tab from the right on your very own GitHub page. I can understand if it didn't ring any bells; it's not very voluminous.

I make a point to look at those, because they often contain critical information like "You will be needing (a lot of) block/item X to access some/most of the mod".

> No matter what source you use, that recipe is displayed in exclusively diorite. Even JEI will say "accepts any 'stone'" for any of the recipes that accept any of the vanilla stones -- it won't say that for this recipe.

There seems to be a critical misunderstanding here. I know the pestle is diorite-only,, I questioned the exclusivity of the recipe. However, you've said this is a creative decision you won't go back on, so that conversation is done.

Nevertheless, please consider adding a line about needing those stone types on that wiki.

**Comment by elucent** [comment]
The wiki on the wiki page isn't an actual source of documentation. I'll add a line to discourage anyone from ever trying to use it. Use the in-game docs instead.

**Comment by Mythrrinthael** [comment]
How is anyone supposed to access the tablet without making a world?

**Comment by Fortanono** [comment]
Try changing it with CraftTweaker. Seriously, this is ridiculous.

**Comment by Mythrrinthael** [comment]
>Try changing it with CraftTweaker. Seriously, this is ridiculous.

I see you, too, missed the point entirely. I got my hands on diorite before I made my second reply through one of a handful of recipes that was added by a very minor mod; didn't even need CraftTweaker.
It wasn't about _me_ anymore.

I kept going at this thread as long as I did because I was baffled that he doesn't know, or perhaps doesn't care for, the concept of "common courtesy" and would rather have this conversation again with someone else than adding _just three words_ on a webpage he can freely edit.. Seriously, talking about "not my responsibility" instead of exerting truly microscopic effort?
